Create fiche_projection.qmd

Nigilan-rgb opened the pull request:
J'ai fait une fiche synthétique pour expliquer la projection sur modele de regression multiple

Nigilan-rgb committed:
Create fiche_projection.qmd

J'ai fait une fiche synthétique pour expliquer la projection sur modele de regression multiple

pbastide commented:
Bonjour,
Merci pour cette fiche de synthèse.
Le sujet de la fiche semble utile. L'idéal serait que cette fiche puisse être utilisée par les autres étudiant·e·s.
Pour qu'elle puisse être partagée plus facilement, il pourrait être utile d'ajouter un peu de contexte, par exemple :

* un titre plus informatif (actuellement, "fiche")
* des sous-titres pour les sections
* éventuellement, un graphique pour montrer la projection

N'hésitez pas à amender la fiche en faisant des commits sur la branche `patch-3` que vous avez utilisée pour le PR. Ces commits apparaîtrons automatiquement ici.

Nigilan-rgb committed:
Add files via upload

Nigilan-rgb committed:
Update fiche_projection.qmd

Nigilan-rgb committed:
Update fiche_projection.qmd

j'ai ajouter la source de l'image , et les cite start je pense quil sont apparru apres que j'ai eu du mal a compiler et j'ai demander a gémini de me corriger
